src/univalue/test/object.cpp

```diff
@@ -1,5 +1,5 @@
 // Copyright (c) 2014 BitPay Inc.
-// Copyright (c) 2014-2016 The Bitcoin Core developers
+// Copyright (c) 2014-2017 The Bitcoin Core developers
```

Comment:
This file shouldn't be modified, as it's part of the univalue subtree. I've updated our copyright script in #12075.

Reply:
Edited, thanks!

Reply:
#12075